web/libs/editor/src/tags/object/Paragraphs/Paragraphs.module.scss

```diff
@@ -7,12 +7,35 @@ $border-thin: 1px solid rgb(137 128 152 / 16%);
   margin-right: var(--spacing-tight);
   margin-bottom: 10px;
   display: table;
+  position: relative;
 
   &.numbered::before {
     counter-increment: phrase;
     content: counter(phrase);
     margin-right: 4px;
   }
+
+  // Only add padding when select all button is present
+  &.withSelectAllButton {
+    padding-right: 50px !important; // Override any other padding from newUI
+  }
+  
+  // Ensure newUI doesn't add extra padding unless select all button is present
+  &.newUI {
+    // Don't add any extra padding by default
+    &:not(.withSelectAllButton) {
+      padding-right: 0.5em !important; // Reset to default, override any inherited padding
+    }
+    
+    &.withSelectAllButton {
+      padding-right: 50px !important; // Only add extra padding when button is present
+    }
+  }
+}
```

Comment:
this all seems a little excess, you add this padding later in NewUI and it looks correct